hack/lib/terraform-context.sh

```diff
@@ -2,6 +2,4 @@ terraform_dir="$SCRIPT_DIR/../$terraform_context"
 
 export ASSUME_ROLE=$(terraform -chdir=$terraform_dir output -raw iam_role_arn)
 
-TEMP='/tmp/eks-workshop-shell-env'
-
-terraform -chdir=$terraform_dir output -raw environment_variables > $TEMP
+terraform -chdir=$terraform_dir output -raw environment_variables > ${TEMP:-/tmp}/eks-workshop-shell-env
```

Comment:
Can you explain what drives the need for an explicit TEMP variable for finch specifically?

Reply:
I was using old version of finch and was not reading from /tmp, now is working